.env

```diff
@@ -32,6 +32,12 @@ GETH_BOOTNODE_DOCKER_IMAGE_TAG="master"
 CELOTOOL_DOCKER_IMAGE_REPOSITORY="gcr.io/celo-testnet/celo-monorepo"
 CELOTOOL_DOCKER_IMAGE_TAG="celotool-dfdc3e8b26e98aa294b27e2b5621c184488a10db"
 
+CELOCLI_STANDALONE_IMAGE_REPOSITORY="gcr.io/celo-testnet/celocli-standalone"
+# TODO (yerdua): Change this to something actually reasonable
+CELOCLI_STANDALONE_IMAGE_TAG="yerduaoracletest"
```

Comment:
This really doesn't seem ideal. I needed a docker image that can run the cli. The existing dockerfile has deeply hardcoded references to alfajores and runs a full node. In order to get *something* here, I built an image and pushed it manually.

Reply:
Could you add both env vars to all .env files?